.github/scripts/sheriff-queue.sh

```diff
@@ -0,0 +1,20 @@
+date_string=$(sed '1!d' ./.github/data/sheriff-order)
+start_date=$(date --date=$date_string)
+count=1
+total_records=$(wc -l <./.github/data/sheriff-order)
+let "total_records--"
+while [ $(date --date="$start_date" +%s) -lt $(date +%s) ]; do
+  DOW=$(date -d "$start_date" +%u)
+  if [ \( $DOW -eq 4 \) -o \( $DOW -eq 5 \) ]; then
+    start_date=$(date -d "$start_date+4days")
```

Comment:
Can you please explain why we are adding 4 and 2 days in this if?

Reply:
@jiripetrlik ,
The sheriff role is being assigned to a person for 2 days.

Person1 : Week1 - Monday
Person2 : Week1 - Wednesday
Person3 : Week1 - Friday
Person4 : Week2 - Tuesday
Person5 : Week2 - Thursday

There are two cases here to add 4 days:
1. When a person starts sheriff role on Friday, the next person starts on Tuesday (4 days after)
2. A person starts sheriff role on Thursday, the next person starts on Monday (4 days after)
Other cases we would need to add 2 days.

hope this explains you.